src/h5web/visualizations/heatmap/ColorBar.tsx

```diff
@@ -3,10 +3,15 @@ import { AxisRight } from '@vx/axis';
 import { useMeasure } from 'react-use';
 import { adaptedNumTicks, generateCSSLinearGradient } from './utils';
 import styles from './HeatmapVis.module.css';
-import { useHeatmapState } from './store';
+import {
+  useHeatmapStore,
+  dataScaleSelector,
+  interpolatorSelector,
+} from './store';
 
 function ColorBar(): JSX.Element {
-  const { dataScale, interpolator } = useHeatmapState();
+  const interpolator = useHeatmapStore(interpolatorSelector);
+  const dataScale = useHeatmapStore(dataScaleSelector);
```

Comment:
Properties that were `computed` in easy-peasy needs to be computed from selectors in zustand. There is also the custom hook possibility.

Reply:
In terms of naming, just thinking we could also go for `selectInterpolator`. I'm fine with either.